### PR TITLE
(2645) Add ISPF ODA level C/D bulk upload

### PR DESCRIPTION
**Pull request by yndajas**
## Changes in this PR

This adds level C/D ISPF ODA bulk upload functionality

### Before

image

image

### After

image

image

## Next steps

- [ ] Is an ADR required? An ADR should be added if this PR introduces a change to the architecture.
- [x] Is a changelog entry required? An entry should always be made in `CHANGELOG.md`, unless this PR is a small tweak which has no impact outside the development team.
- [ ] Do any environment variables need amending or adding?
- [ ] Have any changes to the XML been checked with the IATI validator? See [XML Validation](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/doc/xml-validation.md)

### CODE REVIEW

**Comment by yndajas** [comment]
[Add new CSV bulk upload for level C & D ODA ISPF activities](https://trello.com/c/T3fIXAz5/2645-add-new-csv-bulk-upload-for-level-c-d-oda-ispf-activities)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1136,6 +1136,7 @@ activity and on its child transactions (which can be actuals, refunds, and adjus
 ## [unreleased]
 
 - Add Level B ISPF ODA and non-ODA activity bulk uploads
+- Add Level C ISPF ODA activity bulk uploads
```
Comment (CristinaRO) [inline CHANGELOG.md:1139]: The tiniest thing, but should this say "C and D"?

File: app/views/shared/activities/uploads/_upload_form.html.haml [hunk 2]
```diff
@@ -7,7 +7,7 @@
         hint: { text: t("form.hint.activity.csv_file_recover_from_error_html", link: send(path_helper, instance, { type: type, format: :csv })) }
     - else
       = f.govuk_file_field :activity_csv,
-        label: { text: t("form.label.activity.csv_file"), hidden: !local_assigns[:show_hint] },
+        label: { text: t("form.label.activity.csv_file"), hidden: true },
```
Comment (CristinaRO) [inline app/views/shared/activities/uploads/_upload_form.html.haml:10]: If it's always hidden, do we still need it?

**Review by CristinaRO — APPROVED** [review]
Just a couple of comments, one of which a typo that should be fixed. Great work!

File: app/views/shared/activities/uploads/_upload_form.html.haml (hunk 2 above)
Comment (yndajas) [reply app/views/shared/activities/uploads/_upload_form.html.haml:10]: I've kept it in as it's still used by screen readers even when hidden. Maybe the context is explicit enough, though

**Pull request merged**